content/docs/guides/responsive_amp/responsive_design@ko.md

```diff
@@ -0,0 +1,173 @@
+---
+$title: 반응형 AMP 페이지 만들기
+---
+[TOC]
+
+## 소개
+
+반응형 웹 디자인의 목적은 사용자의 요구에 맞는, 즉 기기의 화면 크기와 방향에 따라 반응하는 매끄러운 웹페이지를 만드는 것입니다.  AMP를 사용하면 반응형 페이지를 쉽게 만들 수 있습니다.  AMP는 모든 화면과 기기 카테고리를 지원하며 반응형 구성요소를 기본으로 제공합니다.
+
+이 가이드에서는 AMP에서 필수 반응형 요소를 간편하게 구현하는 방법을 설명합니다.
+
+- [표시 영역 제어](#표시-영역-제어)
+- [반응형 레이아웃 만들기](#반응형-레이아웃-만들기)
+- [미디어 조정](#페이지에-맞게-미디어-조정)
+
+## 표시 영역 제어
+
+콘텐츠가 모든 기기의 브라우저 창에 맞게 조정될 수 있도록 웹페이지를 최적화하려면 `meta` 표시 영역 요소를 지정해야 합니다. 표시 영역 요소는 웹페이지의 표시 영역 비율과 크기를 조정하는 방법을 브라우저에 알려줍니다.
+
+그렇다면 어떤 값을 사용해야 할까요?  AMP에 이미 준비되어 있습니다.  AMP 페이지에 있는 [필수 마크업](/ko/docs/reference/spec.html#required-markup)의 일부로서 다음 표시 영역을 지정해야 합니다.
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+이 설정은 반응형 사이트에 사용하는 일반적인 표시 영역 설정입니다. 올바른 AMP 페이지를 위해 반드시 `initial-scale=1`로 지정해야 하는 것은 아니지만 페이지가 처음 로드될 때 확대/축소 수준이 1이 되도록 이 설정을 유지하는 것이 좋습니다.
+
+## 반응형 레이아웃 만들기
+
+반응형 디자인에서는 CSS [`@media`](https://developer.mozilla.org/docs/Web/CSS/@media) 쿼리를 사용하여 페이지의 콘텐츠를 변경하지 않고도 다양한 화면 크기에 맞게 웹페이지의 스타일을 조정할 수 있습니다.  AMP에서 동일한 CSS `@media` 쿼리를 계속 사용할 수 있습니다. 또한 요소의 `media` 속성을 지정하여 AMP 요소를 더욱 세밀하게 제어할 수 있습니다. 이 기능은 미디어 쿼리에 따라 요소를 표시하거나 숨겨야 할 때 특히 유용합니다. `media` 속성을 사용하는 예를 보려면 [이미지의 아트 디렉션 변경](#changing-the-art-direction-of-an-image) 섹션을 참조하세요.
```

Comment:
Translate `#changing-the-art-direction-of-an-image` -> Should correspond to header within this doc.